cfpb/cfgov-refresh
# Ask CFPB: Add breadcrumbs

Add breadcrumbs to Ask CFPB pages

## Additions

- Add `breadcrumb_items` to Ask page contexts

## Changes

- Update `breadcrumbs.html` and base templates to accommodate Ask `breadcrumb_items`

## Testing

1. Check out branch
2. Set up Ask if necessary
3. Navigate to [category page](http://localhost:8000/ask-cfpb/category-auto-loans/), [student page](http://localhost:8000/ask-cfpb/audience-students/), [search results](http://localhost:8000/ask-cfpb/search/?q=consumer&btnSearch=), and [question page](http://localhost:8000/ask-cfpb/what-effect-will-shopping-for-an-auto-loan-have-on-my-credit-en-763/) and verify breadcrumbs

## Screenshots

<img width="1290" alt="screen shot 2017-06-08 at 8 57 31 am" src="https://user-images.githubusercontent.com/778171/26938585-70a067d8-4c29-11e7-9691-f25ae36136cf.png">
<img width="1223" alt="screen shot 2017-06-08 at 8 57 42 am" src="https://user-images.githubusercontent.com/778171/26938586-7248b23e-4c29-11e7-8aa4-4cd928b57022.png">


## Checklist
